Next generation sequencing (NGS)
################################
:date: 2016-09-14 17:01:45
:author: Nathan Beka
:category: Seminars
:tags: Genomics
:slug: next-generation-sequencing-ngs
:summary: Nathan Beka's journal club session where he summarises his research on Next Generation Sequencing (NGS).

My talk will be a summary of my research so far in investigating the effects of
artefacts which occur during the library preparation stage of Next generation
sequencing.

Next generation sequencing (NGS) of DNA has dramatically transformed approaches
to genomic and genetic research. DNA sequencing refers to a laboratory method
used to determine the sequence of a DNA molecule. Some of the well-known
technologies that are applied in this process include the Roche GS-FLX 454
Genome Sequencer (originally 454 sequencing), the Illumina Genome Analyser
(originally Solexa technology), the ABI SOLiD analyser, Polonator G.007 and the
Helicos HeliScope platforms. These technologies (also referred to as massively
parallel sequencing technologies) have enabled the sequencing of DNA at
unprecedented speeds compared to the "original" sequencing methodology known as
the Sanger method.

Although NGS has revolutionised biology by increasing current understanding of
many genes and genomic regions involved in the pathogenesis of human diseases,
there are challenges associated with the use of these new technologies. For
example, the sequencing of parts of a genome characterized by extremely biased
base composition is still a great challenge to the currently available NGS
platforms. The genomes of certain important pathogenic organisms like
Plasmodium falciparum and Escherichia coli display extremes of base
composition, high AT content and high GC content respectively. The sequencing
"coverage" of these genomes can be affected by artefacts that may be introduced
at various stages of the sequencing process, thus affecting final sequencing
output.

My project studies the effects of artefacts that occur during the initial stage
of the sequencing work flow referred to as library preparation. The purpose of
this research is to identify known artefacts that can occur during the library
preparation stage from existing literature, and to analyse the extent to which
sequencing coverage may be affected by these artefacts. In other words, a study
of problems that can arise from the library preparation stage of the sequencing
work flow and how they affect final sequencing output forms the basis of my
work.

- `Oyola, S.O., Otto, T.D., Gu, Y., Maslen, G., et al. (2012) Optimizing Illumina
  next-generation sequencing library preparation for extremely AT-biased genomes.
  BMC genomics. <http://www.biomedcentral.com/1471-2164/13/1>`__

- `Sanger, F., Nicklen, S. & Coulson, A.R. (1977) DNA sequencing with
  chain-terminating inhibitors. Proceedings of the National Academy of Sciences
  of the United States of America. <http://www.pnas.org/content/74/12/5463.short>`__

- `Zhang, J., Chiodini, R., Badr, A. & Zhang, G. (2011) The impact of
  next-generation sequencing on genomics. Journal of genetics and genomics = Yi
  chuan xue bao. <http://www.sciencedirect.com/science/article/pii/S1673852711000300>`__


**Date:** 16/09/2016 |br|
**Time:** 16:00 |br|
**Location**: LB252


.. |br| raw:: html

    <br />
